v2/googlecloud-to-neo4j/src/main/resources/schemas/connection.schema.json

```diff
@@ -0,0 +1,147 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://cloud.google.com/dataflow/googlecloud-to-neo4j/connection.schema.json",
```

Comment:
not sure what I should put here @bvolpato

Reply:
We're deferring the ownership of the schema to Neo4j (and linking https://neo4j.com/docs/dataflow-bigquery/job-specification in our docs, for example)

Can you host this somewhere publicly?

Reply:
Let me come back to you on that.